test/cctest/test_node_postmortem_metadata.cc

```diff
@@ -87,49 +88,53 @@ TEST_F(DebugSymbolsTest, HandleWrapList) {
   const Argv argv;
   Env env{handle_scope, argv, this};
 
-  uv_handle_t handle;
-
-  node::tracing::TraceEventHelper::SetTracingController(
-      new v8::TracingController());
+  uv_tcp_t handle;
 
   auto obj_template = v8::FunctionTemplate::New(isolate_);
   obj_template->InstanceTemplate()->SetInternalFieldCount(1);
 
   v8::Local<v8::Object> object =
       obj_template->GetFunction()->NewInstance(env.context()).ToLocalChecked();
-  auto* obj = new TestHandleWrap(*env, object, &handle);
+  TestHandleWrap obj(*env, object, &handle);
 
   auto queue = reinterpret_cast<uintptr_t>((*env)->handle_wrap_queue());
-  auto head = queue + nodedbg_offset_HandleWrapQueue__head_;
-  auto next = head + nodedbg_offset_ListNode_HandleWrap__next_;
+  auto head = queue +
```

Comment:
Tiniest of nits but can you break after the `=` instead of the `+`?  (Also below.)